packages/orbit-components/src/HorizontalScroll/HorizontalScroll.stories.js

```diff
@@ -24,10 +25,52 @@ export const Default = (): React.Node => {
           height="full"
         >
           <Text size="large" weight="bold" as="div">
-            <div style={{ height: "50px", display: "flex", alignItems: "center" }}>{key}</div>
+            <div
+              style={{
+                height: "50px",
+                display: "flex",
+                alignItems: "center",
+                scrollSnapAlign: "center",
```

Comment:
If I'm not mistaken, `scrollSnapAlign` doesn't do anything for this story because scroll snapping is not enabled, no?

Reply:
yes, forgot to remove